### PR TITLE
Family page: Add agreement checkmark drawing animation

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Adds checkmark animation to agreement image for Super Official Tech Agreement

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/12111

## Testing

When you enter the agreement section, checkmarks should appear on the browser component. 

This animation should not appear for users who prefer reduced motion.

http://localhost:8000/en-US/firefox/family/#agreement

### CODE REVIEW

**Comment by maureenlholland** [comment]
ready for code review, waiting for design approval before merge

File: media/js/firefox/family/animate-checkmark.es6.js
Comment (maureenlholland) [inline media/js/firefox/family/animate-checkmark.es6.js:36]: on the fence about two things here:

1. in general, should we keep intersection observer isSupported function in a shared file? (will be used across pop-in, checkmark, and other animations)

2. for agreement checkbox drawing, should we avoid all animation if user prefers reduced motion? this one seemed OK to keep even when reduced motion preferred, feels like a fun, subtle touch

Comment (alexgibson) [reply media/js/firefox/family/animate-checkmark.es6.js:36]: As long as each JS script is in the same bundle, seems like an easy enough thing for each function to import?

**Comment by maureenlholland** [comment]
design request for a significant change, moving this out of review ready status

request: have each box containing the checkmark pop in, then the checkmark draws itself

**Review by craigcook — APPROVED** [review]
LGTM! ✔️

**Pull request merged**
